packages/app/prisma-utils/package.json

```diff
@@ -20,7 +20,7 @@
         }
     },
     "scripts": {
-        "build": "rimraf dist && npm run db:update-client && vite build",
+        "build": "rimraf dist && npm run db:update-client && tsc",
```

Comment:
I guess we don't need vite here, but please correct me if it is not true 🙏

Reply:
yeah, I think we don't, it's BE only